scripts/server

```diff
@@ -0,0 +1,13 @@
+#!/usr/local/bin/node
+
+var path = require('path');
+var connect = require('connect');
+var serveStatic = require('serve-static');
+
+var app = connect()
+            .use(serveStatic(path.resolve(__dirname, '../'), {
+                'index': ['Dispatches_v04t.html'],
+            }))
+            .listen(9000, function() {
+                console.log('Server running on 9000...');
+            });
```

Comment:
This is a stylistic preference but since this is only running on our local dev env consider using `const`s and ES6 functions:

```
....
const app = connect();
app.use(serveStatic...);
app.listen(9000, () => { console.log('Server running on 9000') };
```